File: Product.php
<?php

class Product{
    private $idProduct;
    private $price;
    
   

    public function __construct($idProduct = 0, $price = 0) {
        $this->idProduct = $idProduct;
        $this->price = $price;

    }

    public function setIdProduct($idProduct) { //
        $this->idProduct = $idProduct;
    }
    public function getIdProduct() { 
        return $this->idProduct;
    }

    public function setPrice($price) { //
        $this->price = $price;
    }

    public function getPrice(){ 
        return $this->price;

    } 

}
?>

File: JAVA/Product.java
public class Product {
    String idProduk;
    String price;


    Product(){

    }

    Product(String idProduk, String price){
        this.idProduk = idProduk;
        this.price = price;
    }

    void setIdProduk(String idProduk){
        this.idProduk = idProduk;
    }

    void setPrice(String price){
        this.price = price;
    }

    String getIdProduk(){
        return this.idProduk;
    }

    String getPrice(){
        return this.price;
    }

    
}
